server/views/pages/view/cases.njk

```diff
@@ -48,28 +45,24 @@
         </div>
     </div>
 
-    <form method="POST">
```

Comment:
Not sure why this was ever a POST? No data is posted and the handler was just redirecting... can be a GET

Reply:
It might have been a POST redirected to a GET to avoid the back button asking for resubmission of form values? There is a pattern used to avoid these.

Reply:
https://stackoverflow.com/questions/10827242/understanding-the-post-redirect-get-pattern